src/libs/tiny_ttf/lv_tiny_ttf.c

```diff
@@ -141,7 +141,7 @@ static const uint8_t * ttf_get_glyph_bitmap_cb(const lv_font_t * font, uint32_t
     uint32_t stride = lv_draw_buf_width_to_stride(w, LV_COLOR_FORMAT_A8);
     lv_cache_lock();
     uint32_t cp = unicode_letter;
-    lv_cache_entry_t * cache = lv_cache_find(font + cp, LV_CACHE_SRC_TYPE_PTR, 0, 0);
+    lv_cache_entry_t * cache = lv_cache_find(font + cp, LV_CACHE_SRC_TYPE_PTR, font->line_height, 0);
```

Comment:
What do you think about modifying it like this?
```suggestion
    lv_cache_entry_t * cache = lv_cache_find(font, LV_CACHE_SRC_TYPE_PTR, font->line_height, cp);
```

By chance it can happen that `font1+cp_x == font2 + cp_y`.

Reply:
Yes, I agree with you. I have updated the code.